kivy/core/window/_window_sdl2.pyx

```diff
@@ -394,6 +394,21 @@ cdef class _WindowSDL2Storage:
     def set_window_pos(self, x, y):
         SDL_SetWindowPosition(self.win, x, y)
 
+    def set_window_opacity(self, opacity):
+        if SDL_SetWindowOpacity(self.win, opacity):
+            message = (<bytes>SDL_GetError()).decode('utf-8', 'replace')
+            Logger.error(f'WindowSDL: Setting opacity to {opacity} failed - '
+                         f'{message}')
+
```

Comment:
Add `return False` in case of an error and `return True` for success. Append on line 402:
```python
    return False
return True
```

Reply:
Done.